README.md

```diff
@@ -1,5 +1,9 @@
+<<<<<<< HEAD
```

Comment:
@NicholasCorrado: I'm really excited about this PR!

The checked in files have merge conflicts, which sadly you have to resolve by editing this file manually.